msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.targets

```diff
@@ -250,6 +251,7 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 			ExtraArgs="$(CodesignExtraArgs)"
 			IsAppExtension="$(IsAppExtension)"
 			UseHardenedRuntime="$(UseHardenedRuntime)"
+			UseSecureTimestamp="$(UseHardenedRuntime)"
```

Comment:
why a 2nd option ? i.e. when would it make sense to only use one ?

since both are initialized with the same value `UseHardenedRuntime` maybe we should treat this as "notarization" (and not the individual bits) ?

Reply:
I mostly did that in case we wanted to split the functionality. I wasn't sure when else you'd want the timestamp.

Reply:
Timestamps are often used to proof something (signed or not) existed at some point in time. Since Apple modify what's submitted (e.g. bitcode) for iOS it does not have much value.

Having the possibility of setting it separately is not a bad idea :)

> I wasn't sure when else you'd want the timestamp.

Not sure what's the default. Maybe setting it to `none` would be useful (faster) for non-notarized applications (as @rolfbjarne linked to my old experiment, that I totally forgot about)

Reply:
Yeah, I'm reworking it now to be "timestamp if hardened else none".